### DISCUSSION TITLE
An API Component

### DISCUSSION THREAD

**Discussion by Impre-visible** [retypeapp/retype#496] [Ideas]
Hi, I started to use GitBook, but because it's not the best and I discovered retype, I moved to retype, a problem is, there's no premade component for api, with a method badge, a route, the parameters, the responses with code etc... I had to made the components my self with Panel, Badges and Table.
Also, we can't select custom colours for badges, so the green is not great for POST badges.
Hope you will dev that 😄
Thanks

**Comment by geoffreymcgill** [comment]
Ya, no REST API component in Retype, but it is something we would like to implement at some point. Just hard to find the resources available to work on something like that. 

There are `variant` options available for most of the Components, including the `badge`, see https://retype.com/components/badge/#variants. Currently, no options for setting a custom colour though.

Hope this helps.

**Comment by Impre-visible** [comment]
thanks for the reply, I'm gonna wait to the update 😁
